cms/envs/help_tokens.ini

```diff
@@ -12,12 +12,12 @@ updates = course_author:course_assets/handouts_updates.html
 pages = course_author:course_assets/pages.html
 files = course_author:course_assets/course_files.html
 textbooks = course_author:course_assets/textbooks.html
-schedule = course_author:set_up_course/setting_up_student_view.html
+schedule = course_author:set_up_course/studio_add_course_information/index.html
 grading = course_author:grading/index.html
-team_course = course_author:set_up_course/course_staffing.html#add-course-team-members
+team_course = course_author:set_up_course/studio_add_course_information/studio_course_staffing.html
 team_library = course_author:course_components/libraries.html#give-other-users-access-to-your-library
 advanced = course_author:index.html
-checklist = course_author:set_up_course/creating_new_course.html
+checklist = course_author:set_up_course/index.html
```

Comment:
@awaisdar001 Not that I know of. I don't know if there was a test for the original link or if there needs to be one for the new link since I was just replacing the links. I don't know how to write tests, unfortunately. Do you think we need one here?